testplan/web_ui/testing/package.json

```diff
@@ -14,30 +14,46 @@
     "bootstrap": "4.3.1",
     "enzyme": "3.7.0",
     "enzyme-adapter-react-16": "1.6.0",
-    "eslint": "5.10.0",
-    "eslint-plugin-react": "7.11.1",
     "react": "16.6.0",
     "react-copy-html-to-clipboard": "6.0.4",
     "react-custom-scrollbars": "4.2.1",
     "react-dom": "16.6.0",
     "react-portal-tooltip": "2.4.0",
-    "react-scripts": "1.1.1",
+    "react-scripts": "3.0.1",
     "react-spinners": "^0.6.0",
     "react-syntax-highlighter": "^11.0.2",
     "react-test-renderer": "16.6.0",
     "react-vis": "^1.11.7",
     "reactstrap": "6.3.0"
   },
   "devDependencies": {
-    "babel-eslint": "10.0.1",
+    "enzyme-to-json": "^3.4.0",
+    "eslint-plugin-react": "^7.14.3",
     "moxios": "0.4.0"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "test": "react-scripts test",
     "lint": "eslint --ext .js src",
     "eject": "react-scripts eject"
   },
-  "homepage": "/"
+  "homepage": "/",
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
```

Comment:
Why are we opting for these particular versions?

Reply:
These are the default values set by react-scripts 3.